owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.java

```diff
@@ -154,26 +151,21 @@ public void onCreate(Bundle savedInstanceState) {
      * {@inheritDoc}
      */
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         super.onCreateView(inflater, container, savedInstanceState);
         View view = inflater.inflate(R.layout.preview_image_fragment, container, false);
         mProgressBar = view.findViewById(R.id.syncProgressBar);
         DisplayUtils.colorPreLollipopHorizontalProgressBar(mProgressBar);
         mImageView = view.findViewById(R.id.photo_view);
         mImageView.setVisibility(View.GONE);
-        mImageView.setOnClickListener(new OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if(getActivity() != null) {
-                    ((PreviewImageActivity) getActivity()).toggleFullScreen();
-                }
+        mImageView.setOnClickListener(v -> {
+            if(getActivity() != null) {
+                ((PreviewImageActivity) getActivity()).toggleFullScreen();
             }
-
         });
-        mMessageView = view.findViewById(R.id.message);
+        TextView mMessageView = view.findViewById(R.id.message);
```

Comment:
Rename to `messageView`

Reply:
the name comes not from me

Reply:
You already did it via https://github.com/owncloud/android/pull/2411/commits/eed62a59dda8c4dd7665c7f954030dc95930a86b, I know the name doesn't come from you but you changed the scope of it and is not a class field anymore so it should not include `m` as I explained in https://github.com/owncloud/android/pull/2411#discussion_r250082835.